### PR TITLE
feat: added navigation and rating components

### PR DESCRIPTION
**Pull request by sharath-io**
deploy-preview:  https://deploy-preview-6--atom-ui.netlify.app/components/navigation/navigation.html
                              https://deploy-preview-6--atom-ui.netlify.app/components/rating/rating.html

Screen Shot 2022-02-18 at 8 57 45 PM
Screen Shot 2022-02-18 at 8 58 00 PM

### CODE REVIEW

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="author" content="Sharath Edulakanti">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation | Atom</title>
+    <link rel="stylesheet" href="navigation.css">
+    <link rel="stylesheet" href="../../styles.css">
```
Comment (vidhi-thakur) [inline components/navigation/navigation.html:11]: Just a suggestion: you can create a `main` css file where you can import every other css stylesheet and add the link of that stylesheet here

**Review by vidhi-thakur — APPROVED** [review]
Your code looks clean and understandable to me & your design is also simple. Nice :thumbsup:

File: components/navigation/navigation.html (hunk 1 above)
Comment (sharath-io) [reply components/navigation/navigation.html:11]: Thank you for suggestion. Will create a new branch and update  in all components.

Comment (rishbitsnbytes) [reply components/navigation/navigation.html:11]: Yes that's a good suggestion.

**Review by rishbitsnbytes — COMMENTED** [review]
Looks good to merge Sharath!

**Pull request merged**